client-java/sql/src/main/java/org/evomaster/client/java/sql/internal/SqlStringUtils.java

```diff
@@ -0,0 +1,15 @@
+package org.evomaster.client.java.sql.internal;
+
+public class SqlStringUtils {
+    private static final String QUOTE = "'";
```

Comment:
what if the quotation mark is a `"` instead of a `'`?  in any case, should be at least mentioned it here as a comment